### ISSUE TITLE
What's the difference between train_split and proc_split?

### ISSUE THREAD

**Issue by Mike4Ellis** [yuleiniu/cfvqa#9]
I'm confused about why `proc_split` exists.
what will happen if I set `proc_split = 'trainval'` when `split = 'val'`?

**Comment by yuleiniu** [comment]
This setting is only used in VQA v2 dataset. One can use both train and validation sets for training to achieve higher performance on the public leaderboard, which uses the test set for evaluation. 

Since the VQA v2 test set is not used in our experiments, the comments can be ignored. 

You may refer to [block](https://github.com/Cadene/block.bootstrap.pytorch) to learn more about how `proc_split` works.

**Comment by Mike4Ellis** [comment]
So according to my unberstanding, if I just want to do some experiments, the setting `train_split = train, eval_split = val` will work for most of the situation. 
Is that right?

**Comment by yuleiniu** [comment]
Yes, you can fix the `proc_split` setting to do the experiments in our paper.

**Comment by Mike4Ellis** [comment]
Nice, thank you!
